Assets/Scripts/Buddy/BuddyStats.cs

```diff
@@ -212,6 +212,8 @@ public void AdjustHappiness( float deltaHappiness )
 			_happiness = 0;
 		}
 
+		actor.animator.SetFloat( "happiness", _happiness );
```

Comment:
This assumes that it's normalized, right? Because that's not in yet. I can put it in tonight though.